source/postprocess/visualization.cc

```diff
@@ -97,6 +97,36 @@ namespace aspect
             return update_values;
           }
       };
+
+      /**
+       * This Postprocessor will generate the output variables of mesh velocity
+       * for when a free surface is used.
+       */
+      template <int dim>
+      class FreeSurfacePostprocessor: public DataPostprocessorVector< dim >, public SimulatorAccess<dim>
+      {
+        public:
+          FreeSurfacePostprocessor ( const std::string &name, const UpdateFlags update_flags )
+            : DataPostprocessorVector<dim>( name, update_flags )
```

Comment:
since the name and the update flags are always the same, there is no need to take them by argument. Just write a no-argument constructor and put the correct values in where you call the constructor of the base class.